### DISCUSSION TITLE
failed with exitcode 127 : Command not found : sh: 1: mysqldump: not found

### DISCUSSION THREAD

**Discussion by klasenma** [spatie/laravel-backup#1330] [Q&A]
Thanks for this package, it is awesome. I managed to get it work on my local windows machine, but **unfortunately I can't get it to work with laravel vapor on aws s3.**

I don't know the destination of mysqldump on aws and can't find any working example of someone using this package with laravel vapor.

**Comment by marcusaaronb** [comment]
Download MySQL Client to access this features

**Reply by klasenma** [reply]
Hey, thanks for your response. Could you please elaborate? Where should I download the MySql Client, is this something I can do within the aws dashboard?

**Comment by henryejemuta** [comment]
@marcusaaronb I'm having similar issue with DO App Platform, any recommendation?

**Reply by parallels999** [reply]
MySQL Client is needed, you have to install it, or search the bin path and set it on the config file
